projects/demo/src/pages/kit/number/examples/6-custom-minus-sign/mask.ts

```diff
@@ -0,0 +1,9 @@
+import {MaskitoOptions} from '@maskito/core';
+import {maskitoNumberOptionsGenerator} from '@maskito/kit';
+
+export function getMaskitoOptions(minusSign: string): MaskitoOptions {
```

Comment:
Let's do this example in the same way as other examples

Reply:
Let's do